datasets/oneoff/id43005d93-f59a-11ee-8591-ff101ca1cce5_1_0.cgel

```diff
@@ -1,4 +1,71 @@
 # sent_id = id43005d93-f59a-11ee-8591-ff101ca1cce5_1_0
 # text = The Secretary shall establish an advisory committee to be known as the Advisory Committee on Minority Veterans (hereinafter in this section referred to as “the Committee”).
 # sent = the Secretary shall establish an advisory committee to be known as the Advisory Committee on Minority Veterans hereinafter in this section referred to as the Committee
-# tree_by = bwaldon (YYYY-MM-DD)
+# tree_by = bwaldon (2024-11-15)
+(Clause
+    :Subj (NP
+        :Det (DP
+            :Head (D :t "the"))
+        :Head (Nom
+            :Head (N :t "Secretary")))
+    :Head (VP
+        :Head (V_aux :t "shall" :xpos "MD")
+        :Comp (Clause
+            :Head (VP
+                :Head (V :t "establish" :xpos "VB")
+                :Obj (NP
+                    :Det (DP
+                        :Head (D :t "an"))
+                    :Head (Nom
+                        :Head (Nom
+                            :Mod (AdjP
+                                :Head (Adj :t "advisory"))
+                            :Head (N :t "committee"))
+                        :Mod (Clause
+                            :Head (VP
+                                :Marker (Sdr :t "to")
+                                :Head (VP
+                                    :Head (V_aux :t "be" :xpos "VB")
+                                    :Comp (Clause
+                                        :Head (VP
+                                            :Head (V :t "known" :l "know" :xpos "VBN")
+                                            :Comp (PP
+                                                :Head (P :t "as")
+                                                :PredComp (NP
+                                                    :Det (DP
+                                                        :Head (D :t "the"))
+                                                    :Head (Nom
+                                                        :Head (Nom
+                                                            :Mod (Nom
+                                                                :Head (N :t "Advisory"))
```

Comment:
"advisory" above is an Adj (which I think makes more sense than N unless it is "an advisory")